src/Components/Agenda/AgendaLeg/AgendaLeg.jsx

```diff
@@ -433,8 +429,8 @@ const AgendaLeg = props => {
       content: formatVice(leg?.vice),
     },
     {
-      title: 'Pay Plan',
-      content: (<div>{get(leg, 'pay_plan') || defaultSepText || DEFAULT_TEXT}</div>),
+      title: 'PP/Grade',
+      content: (<div>{`${get(leg, 'pay_plan') || defaultSepText || '--'} ${defaultSepText || leg?.grade || '--'}`}</div>),
```

Comment:
```suggestion
      content: (<div>{`defaultSepText || ${get(leg, 'combined_pp_grade') || DEFAULT_TEXT}`}</div>),
```

this should cover it